### PR TITLE
14-invalid-db-file

### PR DESCRIPTION
**Pull request by soerenlemke**

### CODE REVIEW

**Comment by vyshnav-vinod** [comment]
@soerenlemke Does this pass the tests?

File: database/setup_test.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func TestNewDatabase(t *testing.T) {
-	testSampleInvalidDb, _ := filepath.Abs(filepath.Join("testdata", "invaliddb.go"))
+	testSampleInvalidDb, _ := filepath.Abs(filepath.Join("testdata", "invaliddb.db"))
```
Comment (vyshnav-vinod) [inline database/setup_test.go:9]: Hey, what you did here is the `file does not exist test`, which we already did. What we wanted to do is the invalid db file test. Invalid as in the file exists, but is not a sqlite database.

Comment (soerenlemke) [reply database/setup_test.go:9]: I changed the testSampleInvalidDb to be the new invalid .db file. Isnt that right?

Comment (soerenlemke) [reply database/setup_test.go:9]: So can I merge it or am I understanding this completely wrong?

Comment (vyshnav-vinod) [reply database/setup_test.go:9]: But isnt invaliddb.db a non existing file? Or did you forget to push it to the branch?

Comment (soerenlemke) [reply database/setup_test.go:9]: Oh i see... i forgot to add the file to git :)

Comment (vyshnav-vinod) [reply database/setup_test.go:9]: It was probably ignored by gitignore. This seems good. We can merge this now :+1:

**Review by vyshnav-vinod — APPROVED** [review]

File: database/setup_test.go (hunk 1 above)
Comment (soerenlemke) [reply database/setup_test.go:9]: @vyshnav-vinod pushed the file

**Pull request merged**